### PR TITLE
Completed sql-select exercise

### PR DESCRIPTION
**Pull request by techmatlock**

### CODE REVIEW

File: sql-select/actor-forty.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+select *
+from "actors"
+where "actorId" = '40';
```
Comment (shawnkost) [inline sql-select/actor-forty.sql:3]: I'll touch on this in Q&A, but when you are checking for a number, it's better to drop the quotes. Postgres does type coercion (same as JavaScript), so it's converting the string to a number, but not all SQL engines do this, so for consistency & clarity, it'd be better to just do `where "actorId" = 40`

File: sql-select/cheap-to-rent.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+select * from "films"
```
Comment (shawnkost) [inline sql-select/cheap-to-rent.sql:1]: We don't want all the columns in this one, only the specified columns

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just need to fix one of your queries, and left a comment on the other

File: sql-select/actor-forty.sql (hunk 1 above)
Comment (techmatlock) [reply sql-select/actor-forty.sql:3]: Gtk.  Thanks.

**Review by shawnkost — APPROVED** [review]
👍🏻

File: sql-select/actor-forty.sql (hunk 1 above)
Comment (techmatlock) [reply sql-select/actor-forty.sql]: Gtk.  Thanks.

**Pull request merged**